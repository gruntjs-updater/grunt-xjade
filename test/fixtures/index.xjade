exports.render = function @template () {

html
    head
    body
        h1 "Some header text"

}
