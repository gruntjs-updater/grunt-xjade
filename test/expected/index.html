<!DOCTYPE html>
<html>
    
    <head></head>
    
    <body>
        <h1>Some header text</h1>
    </body>

</html>
